__tests__/app.test.js

```diff
@@ -72,16 +72,47 @@ describe('GET:/api/articles/:article_id', () => {
         return request(app)
             .get('/api/articles/scooby')
             .expect(400)
-            .then((response) => {
-                expect(response.body.msg).toBe('bad request');
+            .then(({body}) => {
+                expect(body.msg).toBe('Invalid id type')
             })
     })
-    test('404 - responds with 404 Not Found when given no id',() => {
+})
+describe('GET:/api/articles', () => {
+    test('200 - responds with an array of all articles',() => {
         return request(app)
             .get('/api/articles')
-            .expect(404)
-            .then(({res}) => {
-                expect(res.statusMessage).toBe('Not Found');
+            .expect(200)
+            .then(({body}) => {
+                body.articles.forEach(article => {
```

Comment:
please add a test for the length of the array before using a forEach. If your array is empty the tests inside the array would be skipped but the test for the length wont be skipped.